### DISCUSSION TITLE
Save searches

### DISCUSSION THREAD

**Discussion by marcus-hallenberg** [datalust/seq-tickets#1646] [Help]
Hi

Our compliance chief wants us to be able to log searches (ie if someone makes a search for a specifik ip or individual we want to track this). Is this possible to do?

**Comment by nblumhardt** [comment]
Hi Marcus,

Thanks for getting in touch! It's not currently possible to do this directly within Seq, though we're strongly considering it for inclusion in our next major release.

If you're able to share some more details - would it be sufficient for your use if Seq simply logged this information out to a set of text files on the server, or do you foresee needing to search/slice the information in the Seq UI itself?

Best regards,
Nick

**Comment by marcus-hallenberg** [comment]
Hi

Preferably it would be a thing to do in the UI. But do you see the possibility to do it right now in the form of a file?

The main reason is for us to have a routing where it is possible for GDPR responsible to check if someone is using the logs to get personal information about a user. This is not something that really happens but in order to meet the demands from some ISO standard this requirement should be met.



Marcus Hallenberg
Project Manager
+46 (0)702 46 23 47
***@***.******@***.***>


[Shape  Description automatically generated with medium confidence]<http://www.nexergroup.com/>


[signature_1379867925]<http://www.nexergroup.com/>

**Reply by nblumhardt** [reply]
Thanks for the follow-up. We'll need to dig into options, here - I think using a file could be problematic because of the amount of data generated (many Seq UI interactions generate a lot of separate searches that would need to record detailed payloads - e.g. dashboards, alerts and alert rendering, incremental loading and infinite scrolling of the event search). I'll convert this to an issue so we can track it.

**Reply by nblumhardt** [reply]
https://github.com/datalust/seq-tickets/issues/1669
